src/viam/ur/module/ur_arm.cpp

```diff
@@ -422,9 +422,13 @@ ProtoStruct URArm::do_command(const ProtoStruct& command) {
     constexpr char k_get_tcp_forces_base_key[] = "get_tcp_forces_base";
     constexpr char k_get_tcp_forces_tool_key[] = "get_tcp_forces_tool";
     constexpr char k_clear_pstop[] = "clear_pstop";
+    constexpr char k_arm_ready[] = "arm_ready";
+    constexpr char k_arm_status[] = "arm_status";
```

Comment:
- These are, effectively, method names, so I think they should start with a verb, like `get_` or `is_`.
- We are an arm so I think (hope?) we don't need that in the name
- Maybe `is_[in_]controllable_state` and `get_status_description` or `get_state_description`?